bootstrap.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable import/no-dynamic-require,global-require */
+const fs = require('fs');
+const path = require('path');
+const spawn = require('child_process').spawnSync;
+
+const type = process.env.type || 'upgrade'; // yarn update types
+const logTask = msg => console.log(`👍  ${msg}`);
+
+const installPackage = (at) => {
+  const result = spawn('yarn', [type], { stdio: 'inherit', cwd: at });
+  if (result.error) {
+    console.log(result.error);
+    process.exit(1);
+  }
+  logTask(`installed ${at}\n`);
+};
+
+const packages = fs.readdirSync('packages').reduce((pkgs, pkg) => {
+  const packagePath = path.join(process.cwd(), 'packages', pkg);
+  const packageJSON = path.join(packagePath, 'package.json');
+  try {
+    if (fs.statSync(packagePath).isDirectory() && fs.statSync(packageJSON).isFile()) {
+      pkgs.push({ path: packagePath, name: require(packageJSON).name });
+    }
+  } catch (e) { return pkgs; }
+  return pkgs;
+}, []);
+
+console.log(`\n🔥  Bootstrapping\n`);
+
+// Install the root package.json first so that we can use shelljs.
+installPackage(process.cwd());
+const shell = require('shelljs');
+
+// Install all of the monorepo packages.
+packages.forEach(pkg => installPackage(pkg.path));
+
+// Symlink monorepo package dependencies to local packages.
+packages.forEach((pkg) => {
```

Comment:
This tool is just for local development so you can make changes without having to constantly reinstall node_modules. All published packages will have pinned dependencies.

For example:
I want to add a new utility to kyt-utils to use in a command in kyt-core. When I bootstrap for local development kyt-utils is symlinked in my node_modules for kyt-core and I can make changes directly in my files and see those changes as I run and test kyt-core locally. When I'm ready to commit changes I would update and publish a new version of kyt-utils and then use that pinned version as the dependency of kyt-core. 

Dependencies that we don't manage (eg. jest, shelljs) will always use versions from npm even in local development.